integration

```diff
@@ -541,6 +541,7 @@
   "willholdoway/hifiberry",
   "wimb0/home-assistant-saj-modbus",
   "WolfRevo/climate.spzb0001_thermostat",
+  "xilense/aimp_custom_component"
```

Comment:
This is still not valid JSON

Reply:
yeah sorry, my bad missing comma on JSON

I just updated the JSON